glossary.md

```diff
@@ -160,6 +280,10 @@ Schedulers are components in PD that generate scheduling tasks. Each scheduler i
 - `hot-region-scheduler`: Balances the distribution of hot Regions
 - `evict-leader-{store-id}`: Evicts all leaders of a node (often used for rolling upgrades)
 
+### SST
+
+Static Sorted Table, Sorted String Table, or Sorted Sequence Table (SST) is a file storage format used in RocksDB.
```

Comment:
I don't believe we use the term Sorted Sequence Table in our product. Added a pointer to the rocksdb overview in suggested edit below:

Static Sorted Table or Sorted String Table is a file storage format used in RocksDB (a component used by the [TiKV Storage Engine](rocksdb-overview.md)).